fossology/__main__.py

```diff
@@ -3,4 +3,5 @@
 """
 from .foss_cli import main
 
-main()
```

Comment:
I am not quite sure here. Putting it that way is the recommended way for pypi projects (says https://realpython.com/pypi-publish-python-package/#different-ways-of-calling-a-package) 
But it makes thinking, that __main__.py is executable with python3 - what it not is.

poetry run python3 fossology/__main__.py 
Traceback (most recent call last):
  File "/Users/setzt/20210804_siemens/fossology-python/fossology/__main__.py", line 4, in <module>
    from .foss_cli import main
ImportError: attempted relative import with no known parent package

If we change the import:

from fossology.foss_cli import main

Both approaches:

- poetry run python3 -m fossology
- poetry run python3 fossology/__main__.py

would work.

Reply:
Nope, pytests can't be executed if the `main()` function is executed every time the module is loaded.

I fixed all the errors now and all tests pass.